src/pages/Home/components/TodoItem/TodoItem.tsx

```diff
@@ -182,33 +179,25 @@ export default function TodoItem({ id, description, completed }: Props) {
       </Dialog>
       {isHovered && (
         <>
-          <Tooltip title="Edit" placement="bottom">
-            <IconButton disabled={isDeleting} onClick={handleEditOpen}>
-              <Icon path={mdiPencil} size={1} />
-            </IconButton>
-          </Tooltip>
-          <Box
-            sx={{
-              height: 42,
-              width: 42,
-              display: 'flex',
-              alignItems: 'center',
-              justifyContent: 'center',
-            }}
+          <IconButton
+            tooltipTitle="Edit"
+            tooltipPlacement="bottom"
```

Comment:
As it has the same prop value `bottom` in 2 `IconButton`s, probably in future `IconButton`s it will be the same - `bottom`, so in that case lets do next:
- `tooltipPlacement` should be optional prop
- by default we will set `bottom` value to `tooltipPlacement` (inside the props)
- remove `tooltipPlacement="bottom"` from `IconButton` components, as we have `bottom` by default and we should not repeat